packages/neo-one-client-common/src/common.ts

```diff
@@ -51,6 +51,30 @@ const uInt160ToString = (value: UInt160 | UInt160Hex): string =>
 
 const stringToUInt160 = hexToUInt160;
 
+const stringToUInt160OrECPoint = (stringIn: string): UInt160 | ECPoint | undefined => {
+  let tryUInt160;
+  try {
+    tryUInt160 = stringToUInt160(stringIn);
+  } catch {
+    tryUInt160 = {};
+  }
+  let tryECPoint;
+  try {
+    tryECPoint = stringToECPoint(stringIn);
+  } catch {
+    tryECPoint = {};
+  }
+
+  if (isUInt160(tryUInt160)) {
+    return tryUInt160;
+  }
+  if (isECPoint(tryECPoint)) {
+    return tryECPoint;
+  }
+
+  return undefined;
+};
```

Comment:
@afragapane This is one more function I created that I want to make sure makes sense. It's used when converting a manifest string to a Manifest instance. I created this for the `hashOrGroup` attribute in the `ContractPermissionDescriptor` object. The string could represent a `UInt160` or could be an `ECPoint`. This should work, and I wrote unit tests for it, but I just want another set of eyes on it.

Reply:
What you have works, but I think it could be nice to simplify a little bit by returning inside of the try blocks since they already check that the return is a valid UInt160/ECPoint.  Something like:
```
try {
  const tryUint160 = stringToUInt160(stringIn);
  return tryUint160;
}
```